packages/holocron/__tests__/holocronModule.spec.jsx

```diff
@@ -105,20 +105,20 @@ describe('holocronModule', () => {
   });
 
   describe('executeLoadingFunctions', () => {
-    it('should call setState with success', async () => {
+    it('should call setStatus with success', async () => {
       expect.assertions(1);
```

Comment:
Why is an assertion expected here if we expect `setState` to be successfully called ?

Reply:
executeLoadingFunctions is asynchronous. As per jest documentation
`Expect.assertions(number) verifies that a certain number of assertions are called during a test. This is often useful when testing asynchronous code, in order to make sure that assertions in a callback actually got called.`

I left it as is for that reason.